### PR TITLE
Add to worker the ability to report its version to engine

### PR DESCRIPTION
**Pull request by natabeck**

### CODE REVIEW

**Comment by orius123** [comment]
What is worker version?

**Comment by natabeck** [comment]
In cluster use case with multiple workers we want to know each workers version

**Comment by orius123** [comment]
Score build version?
If not, do it in OO.

File: worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/WorkerVersionServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Created by kravtsov on 07/12/2015
+ */
+
+@Service
+public class WorkerVersionServiceImpl implements WorkerVersionService {
+    @Override
+    public String getWorkerVersion() {
+        return "";
```
Comment (orius123) [inline worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/WorkerVersionServiceImpl.java]: For real?

**Comment by oritstone** [comment]
It should be score version for the use case of score standalone, or in cloudslang, for example.
When embedded in another application, such as OO, they will have their own implementation which can put the application version.

**Comment by orius123** [comment]
Dont really see why cloudslang cares for the worker version.

**Comment by orius123** [comment]
:-1:

**Pull request merged**